### PR TITLE
log_in_log_out

### PR DESCRIPTION
**Pull request by TienTM25**
User log in, log out

Redmine: https://edu-redmine.sun-asterisk.vn/issues/37722

Rubocop:
Screenshot from 2021-06-30 16-54-00

Layout:
Screenshot from 2021-06-30 16-52-06

### CODE REVIEW

**Comment by TienTM25** [comment]
Ready

File: app/assets/stylesheets/login_session.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the LoginSession controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/login_session.scss]: file k dùng xóa đi
1X nhé e

File: app/helpers/login_session_helper.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    current_user.present?
+  end
+
+  def log_out
+    session.delete(:user_id)
```
Comment (ngocvt-0484) [inline app/helpers/login_session_helper.rb]:
```suggestion
    session.delete :user_id```

File: app/assets/stylesheets/login_session.scss (hunk 1 above)
Comment (TienTM25) [reply app/assets/stylesheets/login_session.scss]: Huhu vâng ạ

**Comment by TienTM25** [comment]
Ready

**Pull request merged**